src/e2e-test/java/io/cdap/plugin/cloudsqlpostgresql/stepsdesign/CloudSqlPostGreSql.java

```diff
@@ -95,57 +117,57 @@ public void verifyTheCldMySqlInvalidPrivate() throws InterruptedException {
-            CdfCloudSqlPostGreSqlActions.enterConnectionName(CdapUtils.pluginProp("pSqlConnectionNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterImportQuery(CdapUtils.pluginProp("pSqlImportQuery"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterConnectionName(System.getenv("Cloud_Psql_ConnectionName"));
+            CdfCloudSqlPostGreSqlActions.enterImportQuery(CdapUtils.pluginProp("cloudPsqlImportQuery"));
         } else if (property.equalsIgnoreCase("connectionName")) {
-            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("pSqlReferenceNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("pSqlDatabaseName"));
-            CdfCloudSqlPostGreSqlActions.enterImportQuery(CdapUtils.pluginProp("pSqlImportQuery"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("cloudPsqlDbName"));
+            CdfCloudSqlPostGreSqlActions.enterImportQuery(CdapUtils.pluginProp("cloudPsqlImportQuery"));
         } else if (property.equalsIgnoreCase("importQuery")) {
-            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("pSqlReferenceNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("pSqlDatabaseName"));
-            CdfCloudSqlPostGreSqlActions.enterConnectionName(CdapUtils.pluginProp("pSqlConnectionNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("cloudPsqlDbName"));
+            CdfCloudSqlPostGreSqlActions.enterConnectionName(System.getenv("Cloud_Psql_ConnectionName"));
         } else if (property.equalsIgnoreCase("jdbcPluginName")) {
-            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("pSqlReferenceNameValid"));
+            CdfCloudSqlPostGreSqlActions.enterReferenceName(CdapUtils.pluginProp("cloudPsqlReferenceNameValid"));
             SeleniumHelper.replaceElementValue(CdfCloudSqlPostGreSqlLocators.driverName, "");
-            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("pSqlDatabaseName"));
-            CdfCloudSqlPostGreSqlActions.enterConnectionName(CdapUtils.pluginProp("pSqlConnectionNameValid"));
-            CdfCloudSqlPostGreSqlActions.enterImportQuery(CdapUtils.pluginProp("pSqlImportQuery"));
+            CdfCloudSqlPostGreSqlActions.enterDatabaseName(CdapUtils.pluginProp("cloudPsqlDbName"));
+            CdfCloudSqlPostGreSqlActions.enterConnectionName(System.getenv("Cloud_Psql_ConnectionName"));
+            CdfCloudSqlPostGreSqlActions.enterImportQuery(CdapUtils.pluginProp("cloudPsqlImportQuery"));
         }
```

Comment:
fail test if property is not one of the above values

Reply:
done